docs/standalone-signaling-api-v1.md

```diff
@@ -869,6 +871,98 @@ Message format (Server -> Client):
     }
 
 
+## Internal clients
+
+Internal clients can be used by third-party applications to perform tasks that
+a regular client can not be used. Examples are adding virtual sessions or
+sending media without a regular client connected. This is used for example by
+the SIP bridge to publish mixed phone audio and show "virtual" sessions for the
+individial phone calls.
+
+See above for details on how to connect as internal client. By default, internal
+clients have their "inCall" and the "publishing audio" flags set. Virtual
+sessions have their "inCall" and the "publishing phone" flags set.
+
+This can be changed by including the client feature flag `internal-incall`
+which will require the client to set the flags as necessary.
```

Comment:
The flags need to be set in an explicit `incall` message, and they will be 0 until then, right? Or is it possible to set them already in the `hello` message?

Reply:
They are initially `0` and will need to be set with an `incall` message. I'm rather against including all kinds of optional parameters to the `hello` message.

Reply:
> I'm rather against including all kinds of optional parameters to the hello message.

Totally fine, I just wanted to be sure that I was getting it right :-)